dask/optimization.py

```diff
@@ -236,7 +236,7 @@ def inline(dsk, keys=None, inline_constants=True, dependencies=None):
     Examples
     --------
 
-    >>> d = {'x': 1, 'y': (inc, 'x'), 'z': (add, 'x', 'y')} # doctest: +SKIP
+    >>> d = {'x': 1, 'y': (inc, 'x'), 'z': (add, 'x', 'y')}
     >>> inline(d)       # doctest: +SKIP
```

Comment:
You could unskip this using a similar pattern to the one I described in `cull`

Reply:
Done.